app/models/riak-node.js

```diff
@@ -0,0 +1,36 @@
+import DS from 'ember-data';
+
+var RiakNode = DS.Model.extend({
+  /**
+   * Cluster for which this node belongs to
+   * @property cluster
+   * @type cluster
+   */
+  cluster: DS.belongsTo('cluster'),
+
+  /**
+   * Whether or not the node is available when pinged.
+   *
+   * @property available
+   * @type Boolean
+   */
+  available: DS.attr('boolean'),
+
+  /**
+   * Details on any kind of error if the node is down
+   *
+   * @property errorMessage
+   * @type String
+   */
+  errorMessage: DS.attr('string'),
+
+  /**
+   * The riak node id
+   *
+   * @property nodeId
+   * @type String
+   */
+  nodeId: DS.attr('string')
```

Comment:
This should be omitted. The riak node id should just be Ember's node.id.
No need for a separate attribute for it.